sdk/test/trace/tracer_provider_test.cc

```diff
@@ -12,10 +12,10 @@ TEST(TracerProvider, GetTracer)
 {
   std::shared_ptr<SpanProcessor> processor(new SimpleSpanProcessor(nullptr));
 
-  TracerProvider tf(processor);
-  auto t1 = tf.GetTracer("test");
-  auto t2 = tf.GetTracer("test");
-  auto t3 = tf.GetTracer("different", "1.0.0");
+  TracerProvider tf1(processor);
```

Comment:
It looks like `tf` was originally used to represent `TracerFactory`, probably doesn't make sense now?